### DISCUSSION TITLE
マルチプレイ対応について

### DISCUSSION THREAD

**Discussion by TwilightWings** [pixiv/vroid-sdk-developers#7] [Q&A] [answered]
マルチプレイの対応について以前下記のURLのようなスレッドがありました。
https://spectrum.chat/vroid-developers/general/~57cba8de-352e-41e7-885d-7a82ecce9849

現在はどのような方法でマルチプレイ対応のSDKを入手できますでしょうか？

**Comment by TwilightWings** [comment, marked as answer]
特に回答をいただけないため、独自の方法で対応いたします。

**Reply by Haresaku** [reply]
マルチプレイ対応は、ユーザーが利用できないモデルを描画のために呼び出す必要があり、より慎重な対応が必要だと考えています。

マルチプレイSDKは、シングルプレイの実装の様子含め、具体的な用途を伺い、NDAを締結したうえでお渡ししています。
確認可能なシングルプレイSDK組込み済みのアプリケーションを添え、どういう用途を予定しているか具体的に記述のうえ、以下問い合わせフォームに連絡をお願いいたします。
https://www.pixiv.net/support?mode=inquiry&type=43&service=vroid-sdk

お手数おかけしますが、みなさまが時間をかけて制作した3Dモデルが意図しない形で利用されることを防ぐため、ご理解とご協力のほどよろしくお願いいたします。

**Reply by TwilightWings** [reply]
ご返信ありがとうございます。
問い合わせフォーム利用させていただきます。
